src/Wallabag/CoreBundle/Controller/ConfigController.php

```diff
@@ -173,6 +176,38 @@ public function indexAction(Request $request)
             return $this->redirect($this->generateUrl('config') . '#set5');
         }
 
+        // handle ignore origin rules
+        $ignoreOriginUserRule = new IgnoreOriginUserRule();
+        $action = $this->generateUrl('config') . '#set6';
+
+        if ($request->query->has('ignore-origin-user-rule')) {
+            $ignoreOriginUserRule = $this->getDoctrine()
+                ->getRepository('WallabagCoreBundle:IgnoreOriginUserRule')
+                ->find($request->query->get('ignore-origin-user-rule'));
+
+            if ($this->getUser()->getId() !== $ignoreOriginUserRule->getConfig()->getUser()->getId()) {
+                return $this->redirect($action);
+            }
+
+            $action = $this->generateUrl('config') . '?ignore-origin-user-rule=' . $ignoreOriginUserRule->getId() . '#set6';
```

Comment:
`ignore-origin-user-rule` can be set as a parameter to `generateUrl()`.

Reply:
I'll take a look, thanks